backend/app/utils.py

```
@@ -592,6 +594,73 @@ def resolve_audio_base64(data: str, mime_type: str) -> tuple[str, str | None]:
+            if length and int(length) > MAX_AUDIO_SIZE:
+                logger.error(
+                    f"Fiile too large: {length} bytes. Upto 50 MB audio files are allowed."
+                )
+                return None, f"File too large : {length} bytes."
+            chunks = []
+            downloaded = 0
+            for chunk in resp.iter_content(chunk_size=8192):
+                downloaded += len(chunk)
+                if downloaded > MAX_AUDIO_SIZE:
+                    logger.error(
+                        f"File size exceeded max size of 50MB during download."
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)